lib/wibox/layout/constraint.lua

```diff
@@ -1,140 +1,18 @@
 ---------------------------------------------------------------------------
+-- This class has been moved to `wibox.container.`
+--
 -- @author Lukáš Hrázký
 -- @copyright 2012 Lukáš Hrázký
 -- @release @AWESOME_VERSION@
 -- @classmod wibox.layout.constraint
 ---------------------------------------------------------------------------
 
-local pairs = pairs
-local type = type
-local setmetatable = setmetatable
-local base = require("wibox.widget.base")
-local math = math
+local util = require("awful.util")
```

Comment:
Sigh. I hate dependencies from wibox into awful, but I guess I'll have to live with it.